CLAUDE.md

```
@@ -411,6 +404,44 @@ OneKey is an open-source multi-chain crypto wallet with a monorepo architecture
 - Avoid using non-English languages in comments to maintain consistency and accessibility for all developers
 - Do not use Chinese comments; always use English comments only
 
+## Git Usage Guidelines
+
+### Branch Management
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)